pkg/components/components.go

```diff
@@ -370,163 +398,30 @@ func GetRole(namespace string) *rbacv1.Role {
-				},
-				Resources: []string{
-					"prometheusrules",
-					"servicemonitors",
-				},
-				Verbs: []string{
-					"get",
-					"create",
-					"update",
-					"delete",
-				},
-			},
-			{
-				APIGroups: []string{
-					"rbac.authorization.k8s.io",
-				},
-				Resources: []string{
-					"roles",
-					"rolebindings",
-				},
-				Verbs: []string{
-					"get",
-					"create",
-					"update",
-					"delete",
-				},
-			},
-			{
-				APIGroups: []string{
-					"policy",
-				},
-				Resources: []string{
-					"poddisruptionbudgets",
-				},
-				Verbs: []string{
-					"get",
-					"delete",
-				},
-			},
-			{
-				APIGroups: []string{
-					"",
-				},
-				Resources: []string{
-					"configmaps",
-				},
-				Verbs: []string{
-					"patch",
-				},
-			},
-			{
-				APIGroups: []string{
-					"coordination.k8s.io",
-				},
-				Resources: []string{
-					"leases",
-				},
-				Verbs: []string{
-					"get",
-					"list",
-					"watch",
-					"create",
-					"update",
-					"patch",
-					"delete",
-				},
-			},
-			{
-				APIGroups: []string{
-					"cert-manager.io",
-				},
-				Resources: []string{
-					"certificates",
-					"issuers",
-				},
-				Verbs: []string{
-					"get",
-					"create",
-					"update",
-					"delete",
-				},
-			},
-		},
+		Rules: rules,
 	}
 	return role
 }
 
 func GetClusterRole(allowMultus bool) *rbacv1.ClusterRole {
+	rules := []rbacv1.PolicyRule{
+		newPolicyRule([]string{"networkaddonsoperator.network.kubevirt.io"}, []string{"networkaddonsconfigs"}, []string{"list", "watch"}),
+		newPolicyRule([]string{"networkaddonsoperator.network.kubevirt.io"}, []string{"networkaddonsconfigs/status"}, []string{"patch"}),
+		newPolicyRule([]string{"networkaddonsoperator.network.kubevirt.io"}, []string{"networkaddonsconfigs/finalizers"}, []string{"update"}),
+		newPolicyRule([]string{ocpoperatorv1.GroupName}, []string{"networks"}, []string{"list", "watch"}),
+		newPolicyRule([]string{ocpsecurityv1.GroupName}, []string{"securitycontextconstraints"}, []string{"get", "list", "create", "update"}),
+		newPolicyRule([]string{extv1.GroupName}, []string{"customresourcedefinitions"}, []string{"get", "create", "update"}),
+		newPolicyRule([]string{appsv1.GroupName}, []string{"deployments", "daemonsets"}, []string{"list", "watch"}),
+		newPolicyRule([]string{kvcore.GroupName}, []string{"virtualmachines/finalizers", "virtualmachineinstances/finalizers"}, []string{"update"}),
+		newPolicyRule([]string{corev1.GroupName}, []string{"configmaps", "namespaces"}, []string{"list", "watch"}),
+		newPolicyRule([]string{corev1.GroupName}, []string{"events"}, []string{"create", "patch"}),
+		newPolicyRule([]string{corev1.GroupName}, []string{"services"}, []string{"delete"}),
+		newPolicyRule([]string{authenticationv1.GroupName}, []string{"clusterroles"}, []string{"get", "create", "update", "bind", "delete"}),
+		newPolicyRule([]string{authorizationv1.GroupName}, []string{"clusterrolebindings"}, []string{"get", "create", "update", "delete"}),
```

Comment:
`authenticationv1.GroupName` => `rbacv1.GroupName`
`authorizationv1.GroupName` => `rbacv1.GroupName`

Reply:
but they are different group names

Reply:
yes, for `tokenreviews` and `subjectaccessreviews`, but not here. 

`clusterroles` and `clusterrolebindings` are part of rbac

Reply:
![image](https://github.com/user-attachments/assets/2acb341e-a676-4f28-a2f8-1cb3e9815845)